apps/wallet-mobile/src/yoroi-wallets/hooks/index.ts

```diff
@@ -960,3 +960,86 @@ export function useHideBottomTabBar() {
     return () => navigation.getParent()?.setOptions({tabBarStyle: true, tabBarVisible: undefined})
   }, [navigation])
 }
+
+const supportedTypes = [
+  'img/png', // Yeah, someone minted that
```

Comment:
👍🏻

Reply:
I don't love to have to list all the supported types here, but it's better than trying to fetch videos, audios, and any other thing we know won't work.
Unfortunately it means wrongly typed things like `img/png` need to be here or won't load.